codegen/render/funcs.go

```diff
@@ -75,6 +76,40 @@ func makeTemplateFuncs(customFuncs template.FuncMap) template.FuncMap {
 			}
 			return t.Name
 		},
+
+		"get_operator_values": func(o values.UserOperatorValues) map[string]interface{} {
+			opValues := map[string]interface{}{
+				strcase.ToLowerCamel(o.Name): o.Values,
+			}
+			if o.ValuePath != "" {
+				splitPath := strings.Split(o.ValuePath, ".")
+				for _, p := range splitPath {
+					opValues = map[string]interface{}{
+						strcase.ToLowerCamel(p): opValues,
+					}
+				}
+			}
+			return fromYAML(toYAML(opValues))
```

Comment:
Is this double conversion just to check if the values are valid yaml?

Reply:
quick and dirty method to get it to `map[string]interface{}` instead of the structs so the built in mergeOverwrite function will work in the template